packages/flutter/lib/src/material/pickers/date_picker_header.dart

```diff
@@ -142,7 +142,7 @@ class DatePickerHeader extends StatelessWidget {
                  children: <Widget>[
                    const SizedBox(height: 16),
                    Flexible(child: help),
```

Comment:
We want to remove the `Flexible` from the help widget with this change. In essence we are making the `SizedBox` flexible and not the help widget. We always want the help widget to be as big as it needs to be, but the `SizedBox` is just used for spacing, so it can be smaller.

If we make them both `Flexible`, then any missing height will be taken from both of them equally instead of just from the spacing `SizedBox`, leaving the help text potentially cut off again.

Reply:
PR updated. Thank you @darrenaustin